persistence/persistence-jdbc/src/main/java/uk/gov/justice/services/jdbc/persistence/JdbcResultSetStreamer.java

```diff
@@ -16,12 +16,12 @@
  */
 public class JdbcResultSetStreamer {
 
-    private static final int FETCH_SIZE = 50;
+    private static final int FETCH_SIZE = 100;
 
     public <T> Stream<T> streamOf(final PreparedStatementWrapper psWrapper, final Function<ResultSet, T> resultSetToEntityMapper) throws SQLException {
 
+        psWrapper.setFetchSize(FETCH_SIZE);
```

Comment:
If the fetch size is a hard coded value, and is being moved from the result set to the prepared statement wrapper, it would be better to hard code the fetch size in prepared statement wrapper and get it to set the fetch size itself

Reply:
I have moved this to psWrapper.executeQuery()